django_lti_tool_provider/models.py

```diff
@@ -0,0 +1,58 @@
+import logging
+
+from django.db import models
+from django.contrib.auth.models import User
+from jsonfield import JSONField
+
+from ims_lti_py.tool_provider import DjangoToolProvider
+from django.conf import settings
+
+
+_logger = logging.getLogger(__name__)
+
+
+class LtiUserData(models.Model):
+    user = models.OneToOneField(User)
+    edx_lti_parameters = JSONField(default={})
```

Comment:
@e-kolpakov I don't quite get it.  If the parameters vary by course, how does it make sense to store them on the user model?